### PR TITLE
About page Milestone 1

### PR DESCRIPTION
**Pull request by hoangviet1993**
This PR sets up the "About" page and attempts to match the mocks. Low number of assets so I got a lot of work done! PTAL @seanlip 
Regular view:
about_page_reg_view
Mobile view (partial screenshot):
screen shot 2018-07-23 at 4 36 01 pm

screen shot 2018-07-23 at 4 36 11 pm

screen shot 2018-07-23 at 4 36 18 pm

### CODE REVIEW

**Comment by hoangviet1993** [comment]
Managed to dig up some assets from @jblair87 's PR. Thanks!

Updated current home page accordingly.

Regular view:
home_page_reg_view

File: pages/about/about.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<section id="about">
+  <md-card layout-fill class="oppia-foundation-vertical-padding oppia-foundation-green-background">
+    <md-card-content layout="column" layout-align="center center" class="oppia-foundation-white-text">
+      <md-content layout-wrap class="oppia-foundation-green-background oppia-foundation-white-text">
+        <center>
+          <img ng-src="/static/images/ _85px_white_mintbkgd.png" style="width: 250px;">
```
Comment (seanlip) [inline pages/about/about.html]: Is there supposed to be a space before the _ ?

Consider renaming the image to something like about_page_logo.png or similar.

File: pages/home/home.html [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
-          <h3 hide-xs hide-sm class="oppia-foundation-horizontal-padding">
+          <h3 hide-xs hide-sm class="oppia-foundation-horizontal-padding md-title">
             Students from every corner of the world deserve access to engaging, personalized education.
           </h3>
-          <p hide-gt-sm style="font-size: 10pt; font-weight: bold;">
-            We're building a global classroom
-          </p>
           <p hide-gt-sm style="font-size: 7pt;">
-            Students from every corner of the world deserve access to engaging, personalized education.
+              Students from every corner of the world deserve access to engaging, personalized education.
```
Comment (seanlip) [inline pages/home/home.html]: Deindent by 2

Comment (hoangviet1993) [reply pages/home/home.html]: Done! I have disabled my VSCode's auto-indent settings to stop this from happening.

File: pages/home/home.html [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
 <section id="main-content">
   <md-card layout-fill style="position: relative;">
     <div id="home-page-welcome-image-container" flex layout="column" layout-fill class="oppia-foundation-clearfix">
-      <img ng-src="https://placekitten.com/3000/1200" class="home-page-welcome-image">
-      <md-card-content layout="column" layout-align="center center" class="home-page-welcome-image-overlay horizontal-padding">
-        <div class="home-page-welcome-text-overlay">
-          <h1 hide-xs hide-sm class="oppia-foundation-horizontal-padding">
-            We're building a global classroom
-          </h1>
+      <img ng-src="/static/images/home_cheering_child.png" class="home-page-welcome-image">
```
Comment (seanlip) [inline pages/home/home.html]: I don't think you need ng-src here, since no part of the image is being interpolated. src will do.

**Comment by seanlip** [comment]
LGTM. Thanks!

File: pages/home/home.html (hunk 3 above)
Comment (hoangviet1993) [reply pages/home/home.html]: Oh wow, good catch there. Thanks!
I probably did this in other html file and need to fix those errors over there too.
Will patch them separately in a different PR!

**Pull request merged**